### PR TITLE
feat(ui/autocomplete): updates ui, updates related icons

### PR DESCRIPTION
**Pull request by naveenvignesh5**
Screenshot 2022-03-05 at 8 09 47 AM

### CODE REVIEW

File: react/src/components/ui/icons/SearchIcon/SearchIcon.js [hunk 1]
```diff
@@ -1,12 +1,39 @@
...
-function SearchIcon() {
+import PropTypes from 'prop-types'
+
+function SearchIcon(props) {
   return (
-    <>
-      <p className={styles.line}>SearchIcon goes here.</p>
-    </>
+    <span
+      style={{
```
Comment (ramsunvtech) [inline react/src/components/ui/icons/SearchIcon/SearchIcon.js:8]: add class no inline style

File: react/src/components/blocks/AutoCompleteSearchBox/AutoCompleteListItem.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+// styles
+import styles from './AutoCompleteSearchBox.module.css'
+
+function AutoCompleteListItem({ item }) {
+  return (
+    <li className={styles.autoCompleteResultListItem}>
+      <div className={styles.autoCompleteResultItem}>
+        <div className={styles.searchIcon}>
+          <SearchIcon size={20} color="#9aa0a6" />
```
Comment (ramsunvtech) [inline react/src/components/blocks/AutoCompleteSearchBox/AutoCompleteListItem.js:15]: get color code from util/pallete.js

File: react/src/components/ui/icons/SearchIcon/SearchIcon.js (hunk 1 above)
Comment (naveenvignesh5) [reply react/src/components/ui/icons/SearchIcon/SearchIcon.js:9]: Done. Kindly check @ramsunvtech.

File: react/src/components/blocks/AutoCompleteSearchBox/AutoCompleteListItem.js (hunk 2 above)
Comment (naveenvignesh5) [reply react/src/components/blocks/AutoCompleteSearchBox/AutoCompleteListItem.js]: Updated. Kindly check. @ramsunvtech.

**Review by ramsunvtech — APPROVED** [review]

**Pull request merged**